pkg/stack/loader.go

```diff
@@ -57,21 +60,41 @@ func LoadStacks(stackService Service) {
 			log.Fatalf("Failed to read stack: %s", err)
 		}
 
-		stackParameters := extractStackParameters(file)
+		hostnamePattern := extractMetadataParameters(file, hostnamePatternIdentifier)
+		if 0 < len(hostnamePattern) && len(hostnamePattern) < 2 {
```

Comment:
I think you mean `len(hostnamePattern) == 1`, right? If so please simplify to that expression 😄 

Should we return an error if `len(hostnamePattern) > 1` ?

Reply:
That is indeed what I mean! Doh! :roll_eyes: 

Yes, we should! However I think for now we should follow the pattern implemented in the rest of the loader and simply use log.Fatalf. After gaining more experience with Go I'm aware it's not ideal. My original thought was that this is happens at boot time and by using log.Fatalf I wanted to ensure it failed early if the stacks couldn't be parsed.